.analysis_options

```diff
@@ -6,13 +6,23 @@ analyzer:
 linter:
   rules:
 #    - always_declare_return_types
+#    - always_put_control_body_on_new_line
+    - always_require_non_null_named_parameters
```

Comment:
I got it, I am going to close this branch and wait for the discussions, we are also are implementing quick-fixes so with the delinte we could try to do it automatically